dot/state/network_test.go

```diff
@@ -29,7 +31,9 @@ import (
 
 var testHealth = &common.Health{}
 var testNetworkState = &common.NetworkState{}
-var testPeers = &[]common.PeerInfo{}
+var testPeers = &[]common.PeerInfo{{PeerID: "ID1", BestHash: common.Hash{}, BestNumber: 1, ProtocolVersion: 2, Roles: 0x03},
+	{PeerID: "ID40", BestHash: common.Hash{}, BestNumber: 50, ProtocolVersion: 60, Roles: 0x70},
+}
```

Comment:
maybe update `PeerID` for each peer to use the peer ids of `alice` and `bob` but not necessary